### PR TITLE
Update contribution guide for MacOS image to prevent customers contributing to MacOS source

### PR DESCRIPTION
**Pull request by maxim-lobanov**
# Description
Since our MacOS CI doesn't support external contributions, we would like to update contribution guide to prevent customers about contributing to MacOS source.

## Check list
- [x] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ New tool, Bug fixing, or Improvement?
 Please include a summary of the change and which issue is fixed. Also include relevant motivation and context.  
 **For new tools, please provide total size and installation time.**
 
+<!-- Currently, we don't accept external contributions to MacOS source. Please find more details in [CONTRIBUTING.md](CONTRIBUTING.md#macOS) guide -->
```
Comment (alepauly) [inline .github/pull_request_template.md:6]:
```suggestion
<!-- Currently, we can't accept external contributions to MacOS source. Please find more details in [CONTRIBUTING.md](CONTRIBUTING.md#macOS) guide -->
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -51,7 +51,8 @@ Add `Invoke-PesterTests -TestFile <testFileName> [-TestName <describeName>]` at
   - Use `DocumentInstalledItem "<add to docs>"` helper for building documentation.
 
 ### macOS
-We are in the process of preparing our macOS source to live in this repo so we can take contributions from the community. Until then, we appreciate your patience and ask you continue to make tool requests by filing issues.
+macOS source lives in this repository and available for everyone. However, macOS image-generation CI doesn't support external contributions yet so we are not ready to accept pull-requests for now.
```
Comment (alepauly) [inline CONTRIBUTING.md:54]:
```suggestion
macOS source lives in this repository and available for everyone. However, macOS image-generation CI doesn't support external contributions yet so we are not able to accept pull-requests for now.
```

File: README.md [hunk 3]
```diff
@@ -19,7 +19,8 @@ The "ubuntu-latest" YAML workflow label still uses the Ubuntu 18.04 virtual envi
 
 ***Looking for other Linux distributions?*** We do not plan to offer other Linux distributions. We recommend using Docker if you'd like to build using other distributions with the hosted virtual environments. Alternatively, you can leverage [self-hosted runners] and fully customize your environment to your needs.
 
-***Where is the macOS source?*** We are in the process of preparing our macOS source to live in this repo so we can take contributions from the community. Until then, we appreciate your patience and ask you continue to make tool requests by filing issues.
+***How to contribute to macOS source?*** macOS source lives in this repository and available for everyone. However, MacOS image-generation CI doesn't support external contributions yet so we are not ready to accept pull-requests for now.
```
Comment (alepauly) [inline README.md:22]:
```suggestion
***How to contribute to macOS source?*** macOS source lives in this repository and available for everyone. However, MacOS image-generation CI doesn't support external contributions yet so we are not able to accept pull-requests for now.

```

**Review by alepauly — APPROVED** [review]
minor nits, thanks @maxim-lobanov !

File: .github/pull_request_template.md (hunk 1 above)
Comment (maxim-lobanov) [reply .github/pull_request_template.md]: Thank you for polishing it, fixed!

**Review by AlenaSviridenko — APPROVED** [review]

**Pull request merged**